dts/bindings/sdhc/zephyr,sdhc-spi-slot.yaml

```diff
@@ -3,3 +3,5 @@ description: Generic Zephyr SPI based SDHC controller
 compatible: "zephyr,sdhc-spi-slot"
 
 include: [spi-device.yaml]
+
+bus: sd
```

Comment:
I don't completely understand the `bus` and `on-bus` properties. Is the `bus: sd` supposed to be in file that is including a file (here: `spi-device.yaml`) that contains `on-bus: spi`?

Reply:
Yes. `on-bus` basically requires that the parent node have a matching `bus` entry. So this change allows children of this node that are `on-bus: sd`, like `sdmmc-disk`.